Automattic/jetpack
# Stats: Update mentions of "Site Stats" with "Jetpack Stats"

#### Changes proposed in this Pull Request:

Replaces "Site Stats" with "Jetpack Stats"

#### Other information:

#### Jetpack product discussion


p1HpG7-icd-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply this change to your Jetpack test site.
* Ensure appropriate mentions of "Site Search" has been replaced within wp-admin on the following pages:
  * Dashboard (At-a-glance)
  * Dashboard -> My Plan
  * Settings -> Traffic
  * Stats
